stm32-modules/include/thermocycler-gen2/thermocycler-gen2/messages.hpp

```diff
@@ -361,6 +361,8 @@ struct CloseLidMessage {
     uint32_t id;
 };
 
+struct LidStepperError {};
```

Comment:
Will this be used?

Reply:
nah, I realized I can just use the existing `ErrorMessage`